engine/schema/src/main/java/com/cloud/network/dao/NetworkDaoImpl.java

```diff
@@ -572,7 +572,7 @@ public List<NetworkVO> listByPhysicalNetwork(final long physicalNetworkId) {
     public List<NetworkVO> listByPhysicalNetworkTrafficType(final long physicalNetworkId, final TrafficType trafficType) {
         final SearchCriteria<NetworkVO> sc = AllFieldsSearch.create();
         sc.setParameters("trafficType", trafficType);
-        sc.setParameters("physicalNetworkId", physicalNetworkId);
+        sc.setParameters("physicalNetwork", physicalNetworkId);
```

Comment:
this is strange, the field in both `NetworkVO` and in `PhysicalNetworkTrafficType` is called `physicalNetworkId`. I would say the new value is the typo. Do you know why this should work this way @GutoVeronezi ?

Reply:
yes, you are right, very confusing, but I shouldn´t rely on naming conventions ;)